addons/hr/models/hr_employee.py

```diff
@@ -72,6 +72,14 @@ class HrEmployee(models.Model):
     )
     versions_count = fields.Integer(compute='_compute_versions_count', groups="hr.group_hr_user")
 
+    contract_template_id = fields.Many2one(
+        'hr.version',
+        string="Contract Template",
+        groups="hr.group_hr_user,hr_payroll.group_hr_payroll_user",
+        domain=lambda self: [('company_id', '=', self.env.company.id), ('employee_id', '=', False)],
+        help="Select a contract template to auto-fill form fields with predefined values.",
+    )
```

Comment:
This field is already defined on ``hr.version`` and thus is inherited by ``hr.employee``.

If the goal here is to override the groups definition, you can only redefine this single attribute, and not the others.